std/stream.d

```diff
@@ -1,8 +1,8 @@
 // Written in the D programming language
 
 /**
- * $(RED Warning: This module is considered out-dated and not up to Phobos'
- *       current standards. It will be removed by phobos version 2.070.)
+ * $(RED Deprecated: This module is considered out-dated and not up to Phobos'
+ *       current standards. It will be remove in October 2016.)
```

Comment:
std.stream is already gone. It was removed with https://github.com/dlang/phobos/pull/4801. I don't know why you're commenting on old code here.